### PR TITLE
improve navigation design

### LINKED ISSUE #780: Improvement design of navigation
**Is your feature request related to a problem? Please describe.**
As discussed with @ashachaf, It is necessary to improve the current navigation design.

- Side navigation takes up a lot of space on the desktop.
- The current bottom sticky bar on the mobile looks good, but if more menus, it needs have options to choose from.

**Describe the solution you'd like**

- A toggle button to shrink or expand space on sidebar. When shrink, only visible menu icons without title of menu.
- Insert the arrow button in the current position of the vote button, and click this button to open a pop-up and view another menu.

**Describe alternatives you've considered**

**Additional context**
Related #730

### PR DESCRIPTION
**Pull request by sirius651**
Fix #780

### CODE REVIEW

**Comment by sirius651** [comment]
Resolved test-mock issue, then we can pass the test while building @Velua

File: tests/unit/__mocks__/helpersMock.ts [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
   return "";
 };
+
+export const compareString = (stringOne: string, stringTwo: string) => {
+  const strings = [stringOne, stringTwo];
+  if (!strings.every(str => typeof str == "string"))
+    throw new Error(
+      `String one: ${stringOne} String two: ${stringTwo} one of them are falsy or not a string`
+    );
+  return stringOne.toLowerCase() == stringTwo.toLowerCase();
```
Comment (Velua) [inline tests/unit/__mocks__/helpersMock.ts:11]: Is this function being re-implemented? What's the go here?

**Review by Velua — CHANGES_REQUESTED** [review]

**Comment by Velua** [comment]
Is it meant to look like this with the Bancor logo sticking out? @ashachaf @sirius651 
https://user-images.githubusercontent.com/8017434/104421400-05420e00-55c7-11eb-8431-6c1f475674a6.mov

**Comment by sirius651** [comment]
> Is it meant to look like this with the Bancor logo sticking out? @ashachaf @sirius651
> https://user-images.githubusercontent.com/8017434/104421400-05420e00-55c7-11eb-8431-6c1f475674a6.mov

https://github.com/bancorprotocol/webapp/issues/780#issuecomment-757435949  as this comment, bancor logo should always be visible. top section should not fold with the menu

File: tests/unit/__mocks__/helpersMock.ts (hunk 1 above)
Comment (sirius651) [reply tests/unit/__mocks__/helpersMock.ts:11]: Ah I'll remove that because I left it in the middle of the resolving test issue.

**Comment by ashachaf** [comment]
https://www.figma.com/file/3yED7o7TgoLwoEjV2EkRLL/Bancor-V2?node-id=1679%3A20
https://www.figma.com/file/3yED7o7TgoLwoEjV2EkRLL/Bancor-V2?node-id=1682%3A25

**Comment by sirius651** [comment]
스크린샷 2021-01-14 오후 12 30 59
스크린샷 2021-01-14 오후 12 31 10

Updated design which more fit the mockup. @ashachaf

**Comment by Velua** [comment]
@ashachaf should we just have white space under the Bancor logo till it hits the menu system?

**Comment by ashachaf** [comment]
white doesnt fit the design.
we can have it on a transparent background that fits the page background.

**Comment by sirius651** [comment]
스크린샷 2021-01-14 오후 5 28 02
스크린샷 2021-01-14 오후 5 28 09



> white doesnt fit the design.
> we can have it on a transparent background that fits the page background.

스크린샷 2021-01-14 오후 5 31 37
스크린샷 2021-01-14 오후 5 31 18

Fixed color and look like this. @ashachaf

**Comment by ashachaf** [comment]
@sirius651 
looks good and smooth.
just on close view, needs to have round corners as in the mock.

**Pull request merged**